src/modules/app/createRouteConfig.js

```diff
@@ -1,47 +1,39 @@
-import SearchPage from 'routes/search/containers/SearchPage'
-import PdfFetcherPage from 'routes/pdf-fetcher/containers/PdfFetcherPage'
-import MainDisclaimerPage from 'routes/main-disclaimer/components/MainDisclaimerPage'
-import CategoriesPage from 'routes/categories/containers/CategoriesPage'
-import ExtrasPage from 'routes/extras/containers/ExtrasPage'
-import DisclaimerPage from 'routes/disclaimer/containers/DisclaimerPage'
-import LandingPage from 'routes/landing/containers/LandingPage'
-import EventsPage from 'routes/events/containers/EventsPage'
 import Route from './Route'
 
 const createRouteConfig = () => [
   new Route({
     path: '/'
   }),
   new Route({
-    id: LandingPage,
+    id: 'landing',
```

Comment:
Is there no better way than hard-coding the id as a string? If not, we should maybe define the ids in a central place in some kind of map and import it whenever we need it.

Reply:
Is there a reason for this change? By using the class as identifier you avoid typos.

Reply:
see pr description

Reply:
But defining these somewhere as constants makes sense to me

Reply:
ok damn need to read the damn description :P

ok makes sense because we used the connected component as id.
Maybe we can think about using flow enums here.

Reply:
See https://flow.org/en/docs/types/utilities/#toc-keys (last example)

Reply:
@Schedulaar Can you do something about this?

Reply:
This only affected the old router config, which has been removed with the last changes of @klinzo :D